dependencies/graknlabs/repositories.bzl

```diff
@@ -56,5 +56,5 @@ def graknlabs_behaviour():
     git_repository(
         name = "graknlabs_behaviour",
         remote = "https://github.com/graknlabs/behaviour",
-        commit = "6764b9ad0bbde211c4511dd0bdb08e07873197af", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_behaviour
+        commit = "7c98d085c8cb93a19c99aefb4bff0b55bef0d836", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_behaviour
```

Comment:
There's a problem in the test definition that we didn't reopen the transaction when transaction throws. We have updated behaviour to fix that in https://github.com/graknlabs/behaviour/pull/128

Reply:
Let's add a test case in BDD to test that transactions are closed when match throws, then update this again.